.circleci/config.yml

```diff
@@ -43,17 +43,12 @@ workflows:
     when:
       and:
         - matches:
-            pattern: "^(?!pull/)[-_./\\w]+$"
+            pattern: "^(?!pull/)[-\\w]+$"
             value: << pipeline.git.branch >>
         - not: << pipeline.parameters.nightly_build >>
         - not: << pipeline.parameters.weekly_build >>
     jobs:
-      - hold:
-          type: approval
-      - apple/authenticate:
-          context: pr-approval
-      - mac_build_and_test:
-          requires: [ hold ]
+      - mac_build_and_test
```

Comment:
Restore changes that were done to try and get non-fork PRs to build but broke main.

Reply:
OK, it looks like this works -- the caveat is you can't name your branch something like `dkoski/scroll`

Reply:
I see so `name1/name2` breaks the pattern (I guess that's true in our other repos as well).